build.rs

```diff
@@ -176,3 +203,17 @@ fn main() -> Result<(), BuildError> {
 
     Ok(())
 }
+
+#[cfg(any(
+    feature = "mkl-static-iomp",
+    feature = "mkl-static-seq",
+    feature = "mkl-dynamic-iomp",
+    feature = "mkl-dynamic-seq"
+))]
+fn suggest_setvars_cmd(root: &str) -> String {
```

Comment:
Great clean up! I think we can just remove this outer cfg block, nothing wrong with having it sit around for all builds.
```suggestion
fn suggest_setvars_cmd(root: &str) -> String {
```